acceptance/ui/features/steps/hosts_steps.rb

```diff
@@ -2,6 +2,18 @@
     visitHostsPage()
 end
 
+When(/^I fill in the Host Name field with the default host name$/) do
+    fillInDefaultHostAndPort()
+end
+
+When(/^I fill in the Resource Pool field with the default resource pool$/) do
+    fillInDefaultResourcePool()
```

Comment:
Similar to last comment, use `fillInResourcePool("default")` instead, and refactor line 22 to call `fillInResourcePool(resourcePool)`